Automattic/jetpack
# VideoPress: only enqueue the bridge script when the player is rendered

Fixes #24765

#### Changes proposed in this Pull Request:

* Moved front end bridge script enqueueing to VideoPress player render functions
* removed duplicate enqueue statement for admin pages (different version number was causing double enqueue)

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


Nope
#### Testing instructions:






* Use a site with VideoPress enabled
* Create a post with a VideoPress shortcode
* Create a post with a `/video` block
* Create a post and paste a VideoPress url in the page (uses oembed)
* Create a post with no VideoPress content in it
* View the pages in the front end, the `videopress-token-bridge.js` script should be rendered only on pages with VideoPress content
* View the homepage, it should also not load the bridge script unless your post summaries include the VideoPress player
* Make one or more of the videos on your posts private
* View the post on the front end to ensure the bridge is working correctly
* Visit `/wp-admin`, private videos in the post editor and the Media Library should still be viewable
